client/scss/main.scss

```diff
@@ -2,6 +2,26 @@
 @import "skeleton";
 @import "leaflet";
 
+body {
+    padding: 50px 0;
+}
+
 #map {
     height: 500px;
+    margin-bottom: 30px;
+}
+
+.container {
+}
+
+.address {
+    text-align: right;
+}
+
+.name {
```

Comment:
there's a good reason to use  `park-name` instead.

http://markdotto.com/2012/02/16/scope-css-classes-with-prefixes/

basically while `.park .name` ensures that we don't effect `.name`  Outside of `.park`  it doesn't prevent conflicts Inside of `.park` eg 
main.css
```
.park .name  {font-weight: bold  }
.something-else .name {color: red }
```

html structure with just classnames
```   
.park
     .name
     .something-else
         .name // element will be styled bold and red
```